store/postgres/src/store.rs

```diff
@@ -845,11 +873,12 @@ impl StoreTrait for Store {
 
         // Prepare the new subscription by creating a channel and a subscription object
         let (sender, receiver) = channel(100);
-        let subscription = Subscription { entities, sender };
 
         // Add the new subscription
         let mut subscriptions = subscriptions.write().unwrap();
-        subscriptions.insert(id, subscription);
+        subscriptions.insert(id, sender);
+
+        let receiver = StoreEventStream::new(receiver).filter_by_entities(entities);
```

Comment:
I'd probably make this the caller's responsibility and drop the `entities` parameter from the method, but to maintain the behavior this makes complete sense. Let's roll with this and move the filtering out later if we deem it appropriate then.

Reply:
Yep, that was exactly what I was thinking there ;)